cmd/lncli/walletrpc_active.go

```diff
@@ -125,16 +132,27 @@ var bumpFeeCommand = cli.Command{
 			Usage: "a manual fee expressed in sat/byte that " +
 				"should be used when sweeping the output",
 		},
+		cli.BoolFlag{
+			Name:  "force",
+			Usage: "sweep even if the yield is negative",
+		},
 	},
 	Action: actionDecorator(bumpFee),
 }
 
 func bumpFee(ctx *cli.Context) error {
 	// Display the command's help message if we do not have the expected
 	// number of arguments/flags.
-	if ctx.NArg() != 1 || ctx.NumFlags() != 1 {
+	if ctx.NArg() != 1 {
 		return cli.ShowCommandHelp(ctx, "bumpfee")
 	}
+	if ctx.IsSet("conf_target") && ctx.IsSet("sat_per_byte") {
+		return errors.New("sat_per_byte and conf_target are " +
+			"mutually exclusive")
+	}
+	if !ctx.IsSet("conf_target") && !ctx.IsSet("sat_per_byte") {
+		return errors.New("must specify a fee preference")
```

Comment:
Aren't these already checked at the RPC level?

Reply:
Yes, that is true. They are. And if they weren't, they should. Removed all this validation code.